sdk/tests/test_filesyncer.py

```diff
@@ -0,0 +1,37 @@
+import os
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+from beta9.sync import IGNORE_FILE_CONTENTS, FileSyncer
+
+
+class TestFileSyncer(TestCase):
+    def test_init_ignore_file(self):
+        syncer = FileSyncer(gateway_stub=MagicMock())
+        syncer._init_ignore_file()
+        self.assertTrue(syncer.ignore_file_path.exists())
+        os.remove(syncer.ignore_file_path)
+
+    def test_read_ignore_file(self):
```

Comment:
nit-pick but I would clean up the spacing a bit on these tests, it's hard to read. otherwise good to go.